scrapi/base/__init__.py

```diff
@@ -169,8 +170,8 @@ def harvest(self, start_date=None, end_date=None):
 
         return rawdoc_list
 
-    def get_records(self, url, start_date, end_date, resump_token=''):
-        data = requests.get(url, throttle=self.timeout)
+    def get_records(self, url, start_date, end_date, verify, resump_token=''):
```

Comment:
I think this function still has access to the class, so you can just use self.verify instead of passing it around.